ksim/task/rl.py

```diff
@@ -1013,15 +1013,21 @@ def get_termination_metrics(self, trajectories: Trajectory) -> dict[str, Array]:
         Args:
             trajectories: The trajectories to get the termination metrics for.
         """
+        termination_timesteps_selected = jnp.where(trajectories.done, trajectories.timestep, 0.0)
+        total_termination_time = jnp.sum(termination_timesteps_selected)
+        total_terminations = trajectories.done.sum()
+        episode_length = jnp.where(
+            total_terminations > 0,
+            total_termination_time / total_terminations,
+            0.0,
+        )
```

Comment:
this is not quite accurate, because `trajectories.timestep[..., 0]` is carried from previous rollout, so it is not zero. also if there are no terminations we should handle this properly. will write a fix